internal/controller/ols_app_server_assets_test.go

```diff
@@ -35,6 +37,24 @@ var _ = Describe("App server assets", func() {
 				Scheme:     k8sClient.Scheme(),
 				stateCache: make(map[string]string),
 			}
+			By("create the provider secret")
+			secret, _ = generateRandomSecret()
```

Comment:
better declare the secret in test function scope at line 24.

Reply:
Updated as suggested.